tests/fixtures/mock.py

```diff
@@ -17,7 +17,12 @@
 # the time it's run the decorators have already been applied to the
 # functions, therefore the patching has no effect
 mp = monkeypatch()
-mp.setattr('django.utils.functional.cached_property', property)
+
+# not sure why this was monkeypatched
```

Comment:
It was a footgun (799a5861). Please take this change out unless you really know you're fixing it (in such case no point in keeping commented code around)